lang/ko.js

```diff
@@ -26,13 +26,13 @@ export default {
 	"components.form-element.defaultError": "{label}이(가) 잘못되었습니다.",
 	"components.form-element.defaultFieldLabel": "필드",
 	"components.form-element.input.email.typeMismatch": "이메일이 유효하지 않습니다.",
-	"components.form-element.input.number.rangeError": "{minExclusive, select, true {{maxExclusive, select, true {Number must be greater than {min} and less than {max}.} other {Number must be greater than {min} and less than or equal to {max}.}}} other {{maxExclusive, select, true {Number must be greater than or equal to {min} and less than {max}.} other {Number must be greater than or equal to {min} and less than or equal to {max}.}}}}",
-	"components.form-element.input.number.rangeOverflow": "{maxExclusive, select, true {숫자가 보다 작음 {max}여야 합니다.} other {숫자가 작거나 같음 {max}여야 합니다.}}",
-	"components.form-element.input.number.rangeUnderflow": "{minExclusive, select, true {숫자가 보다 큼 {min}여야 합니다.} other {숫자가 크거나 같음 {min}여야 합니다.}}",
+	"components.form-element.input.number.rangeError": "{minExclusive, select, true {{maxExclusive, select, true {숫자는 {min}보다 크고 {max}보다 작아야 합니다.} other {숫자는 {min}보다 크고 {max}보다 작거나 같아야 합니다.}}} other {{maxExclusive, select, true {숫자는 {min}보다 크거나 같고 {max}보다 작아야 합니다.} other {숫자는 {min}보다 크거나 같고 {max}보다 작거나 같아야 합니다.}}}}",
+	"components.form-element.input.number.rangeOverflow": "{maxExclusive, select, true {숫자는 {max}보다 작아야 합니다.} other {숫자는 {max}보다 작거나 같아야 합니다.}}",
+	"components.form-element.input.number.rangeUnderflow": "{minExclusive, select, true {숫자는 {min}보다 커야 합니다.} other {숫자는 {min}보다 크거나 같아야 합니다.}",
 	"components.form-element.input.text.tooShort": "{label}은(는) {minlength}자 이상이어야 합니다",
 	"components.form-element.input.url.typeMismatch": "URL이 유효하지 않습니다",
 	"components.form-element.valueMissing": "{label}이(가) 필요합니다.",
-	"components.form-error-summary.errorSummary": "{count, plural, other {제출하신 정보에서 {count}개의 오류가 발견되었습니다}}",
+	"components.form-error-summary.errorSummary": "{count, plural, 1 {제출한 정보에서 1개의 오류가 발견되었습니다} other {제출한 정보에서 {count}개의 오류가 발견되었습니다}}",
```

Comment:
I think the `1` should be `one`

Reply:
@margaree This term needs to use `#` or `{count}`. Otherwise, languages that use the `one` category for numbers besides `1` (including French and Hindi) will display the wrong count.

<img width="432" alt="Screen Shot 2022-05-26 at 9 27 46 AM" src="https://user-images.githubusercontent.com/1289042/170497835-f14b59e7-09b2-4f83-a471-1d04a6e01302.png">

Reply:
Sorry I don't completely follow what change would need to be made there.

The `ko` (and other Asian languages) term now looks like this after code review changes https://github.com/BrightspaceUI/core/pull/2494/files#diff-426802e14623a39688f48248262c84ff748a1b3cdd728354da553bd59a18e769R35
Is this correct or does it need other changes?

The other languages are structured like this https://github.com/BrightspaceUI/core/pull/2494/files#diff-741c65feed340e440eddfb1e933f2ae6f73378bd3e6b4b7c502faf8894c1d99cR35. I tested out French quickly on our form demo page and it seemed to show the correct number/term based on if count was 1 or > 1.